java/tsfile/src/main/java/org/apache/tsfile/read/filter/basic/Filter.java

```diff
@@ -82,6 +82,14 @@ public abstract class Filter {
    */
   public abstract boolean[] satisfyTsBlock(TsBlock tsBlock);
```

Comment:
do we still need this method in IoTDB? if we just to keep compatibility, we can add @Deprecated annotation for this method.